### ISSUE TITLE
Support for playing sound while phone screen is asleep

### ISSUE THREAD

**Issue by pmk65** [watchedcom/watched-js#46]
Sometimes while on the move, I can't watch the screen but still want to be able to listen to the audio,.
Currently the phone (Android) screen have to be awake for sound to play. (This is problematic when having the phone in your pocket 😁 )

Thanks for a great app.

**Comment by maximilize** [comment]
Good point. We'll add an option to allow background playback in one of the next releases.
